cmd/gatekeeper.go

```diff
@@ -0,0 +1,67 @@
+// Gatekeeper is a machine gateway for automatic machine authentication based on cloud provider
+// identity.
+
+package cmd
+
+import (
+	"log"
+	"os"
+
+	"github.com/urfave/cli"
+
+	"github.com/manifoldco/torus-cli/config"
+	"github.com/manifoldco/torus-cli/errs"
+	"github.com/manifoldco/torus-cli/gatekeeper"
+)
+
+func init() {
+	gatekeeper := cli.Command{
+		Name:     "gatekeeper",
+		Usage:    "Manage the machine gatekeeper",
+		Category: "SYSTEM",
+		Subcommands: []cli.Command{
+			{
+				Name:  "start",
+				Usage: "Start the machine gatekeeper",
+				Flags: []cli.Flag{
+					cli.BoolFlag{
```

Comment:
I don't think this is needed anymore. Could I do `config.Load()` below instead since the daemon would already be started?